examples/main.tf

```diff
@@ -520,7 +520,7 @@ resource "instaclustr_cluster" "example-cadenceav-opensearch" {
 
   bundle {
     bundle = "OPENSEARCH"
-    version = "1.3.4"
+    version = "1.3.5"
     options = {
```

Comment:
this seems need fixing node sizes
can you also run it to test?

Reply:
That's right, updated. Testing provision Cadence failed at the cassandra version, and now checking.